src/elements/a-gltf-entity.js

```diff
@@ -83,6 +102,20 @@ const inflateEntities = function(classPrefix, parentEl, node) {
 
   el.setObject3D(node.type.toLowerCase(), node);
 
+  const entityComponents = node.userData;
```

Comment:
We should put them in a sub-key rather than just putting it in the root of userData, since other things may also stick stuff in userData.